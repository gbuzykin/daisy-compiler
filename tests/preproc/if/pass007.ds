#if true
#if true
#info "case #1"
#endif
#elif true
#if true
#info "case #2"
#endif
#endif

#if false
#if true
#info "case #3"
#endif
#elif true
#if true
#info "case #4"
#endif
#endif
