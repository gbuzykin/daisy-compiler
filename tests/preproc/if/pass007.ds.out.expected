./preproc/if/pass007.ds:3:2: info: "case #1"
 3 | #info "case #1"
   |  ^~~~
./preproc/if/pass007.ds:17:2: info: "case #4"
 17 | #info "case #4"
    |  ^~~~
./preproc/if/pass007.ds: info: warnings 0, errors 0
